src/features/metadata-instance-editor/Instance.js

```diff
@@ -626,27 +633,26 @@ class Instance extends React.PureComponent<Props, State> {
                         <LoadingIndicatorWrapper isLoading={isBusy}>
                             <Form onValidSubmit={isDirty ? this.onSave : noop}>
                                 <div className="metadata-instance-editor-instance">
-                                    {isCascadingPolicyApplicable && (
-                                        <CascadePolicy
```

Comment:
this always shows the cascading policy, is that intentional? I assume the cascading policy doesn't apply when we add metadata to files

Reply:
Yeah, it's fine that we're now always rendering the CascadePolicy. Internally, it checks if the user has permissions and shows messages depending on the permission.

![File2023-07-28 at 09 14 33](https://github.com/box/box-ui-elements/assets/16271040/9796ed0c-32e4-4e2e-bc4a-56a5881431c7)

![File2023-07-28 at 09 14 52](https://github.com/box/box-ui-elements/assets/16271040/d91a94ab-edc3-48a9-90c0-5bfe6528db8d)